src/background.js

```diff
@@ -169,34 +150,28 @@ var pb = {
   * @param {String} origin the third party origin to take action on
   */
   saveAction: function(userAction, origin) {
-    var allUserActions = {'block': pb.USER_BLOCK,
-                          'cookieblock': pb.USER_COOKIE_BLOCK,
-                          'allow': pb.USER_ALLOW};
-    pb.storage.setupUserAction(origin, allUserActions[userAction]);
-    pb.log("Finished saving action " + userAction + " for " + origin);
+    var allUserActions = {'block': constants.USER_BLOCK,
+                          'cookieblock': constants.USER_COOKIE_BLOCK,
+                          'allow': constants.USER_ALLOW};
+    this.storage.setupUserAction(origin, allUserActions[userAction]);
+    log("Finished saving action " + userAction + " for " + origin);
 
     // TODO: right now we don't determine whether a reload is needed
     return true;
   },
 
-  /**
-  * reloads a tab
-  * @param {Integer} tabId the chrome tab id
-  */
-  reloadTab: function(tabId){
-    chrome.tabs.reload(tabId);
-  },
 
   /**
   * Populate tabs object with currently open tabs when extension is updated or installed.
   */
   updateTabList: function(){
     // Initialize the tabData/frames object if it is falsey
-    pb.tabData = pb.tabData || {};
+    this.tabData = this.tabData || {};
+    var thesetabs = this.tabData;
```

Comment:
It should be a reference. I assigned `this.tabData` to `thesetabs` so I could access the object inside the closure on the next line. Since `this` is a different thing in the scope of the closure than it is here.